packages/fxa-settings/src/components/FormPasswordWithBalloons/index.tsx

```diff
@@ -183,8 +186,7 @@ export const FormPasswordWithBalloons = ({
               onFocusCb={onFocusMetricsEvent ? onNewPwdFocus : undefined}
               onBlurCb={onNewPwdBlur}
               onChange={() => {
-                trigger(['newPassword', 'confirmPassword']);
-                getValues('confirmPassword') === getValues('newPassword') &&
+                newPassword === confirmPassword &&
```

Comment:
Thanks for the catch. I think there's still a few re-rendering issues with this, but I'll add it for now since it's not related to the fix for this PR. We could probably throw a 2-pointer in the clean up epic to audit our forms for unnecessary rerenders and look further at this then.